sfepy/discrete/fem/meshio.py

```diff
@@ -438,18 +450,31 @@ def write(self, filename, mesh, out=None, ts=None, **kwargs):
             for k in cgrps:
                 idxs = nm.where(cell_groups[cidxs] == k)[0]
                 cell_sets[str(k)].append(cidxs[idxs])
-
         cell_data[cgkey] = cgroups
 
-        meshiolib.write_points_cells(filename, coors, cells,
-                                     point_data=point_data,
-                                     point_sets=point_sets,
-                                     cell_data=cell_data,
-                                     cell_sets=cell_sets,
-                                     file_format=self.file_format)
+        return  coors, cells, point_data, point_sets, cell_data, cell_sets
 
     def read_data(self, step, filename=None, cache=None):
-        m = meshiolib.read(self.filename, file_format=self.file_format)
+        """
+        Renames cell resp. vertex data with name "*:ref"
+        to mat_id resp. node_groups
+
+        Parameters
+        ----------
+        step: has no effect
+        filename: filename to use instead of self.filname
```

Comment:
Thanks, just one typo (self.filname), and maybe use the following:
```
filename : string, optional
    The file name to use instead of self.filename.
```